app/queues/common/expiration-request.server.ts

```diff
@@ -0,0 +1,105 @@
+import { Worker, Queue, UnrecoverableError } from 'bullmq';
+
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { prisma } from '~/db.server';
+
+import { addNotification } from '../notifications/notifications.server';
+import { deleteDnsRequest } from '../dns/dns-flow.server';
+import { RecordType as RecType } from '../notifications/expiration-notification.server';
```

Comment:
Why change the name here?

Reply:
When I tried importing RecordType, I saw that the variable name was not unique; it's used in some Prisma code files and other places. To avoid any future naming conflicts, I thought to rename it.

Reply:
got it.

Reply:
For now, it's not causing a naming conflict, so I have removed the alias.